app/webchat_handlers.py

```diff
@@ -160,15 +164,8 @@ async def post(self, request):
         if not form_valid:
             logger.info('form submission error',
                         client_ip=request['client_ip'])
-            return {
-                'form_value_screen_name': data.get('screen_name'),
-                'form_value_country': data.get('country'),
-                'form_value_query': data.get('query'),
```

Comment:
On error none of the above values are now available to the form. The error page has the values which were not entered previously in the error panel, but the previously entered values have disappeared as well?